.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
```

Comment:
Let's just replace this with https://github.com/github/gitignore/blob/master/Python.gitignore